canonical-websites/www.ubuntu.com
# update /download/cloud/conjure-up to vbt

## Done

* update /download/cloud/conjure-up to vbt
* added color fixes for p-strip--accent fieldset background and border

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/download/cloud/conjure-up)
- compare to the [autopilot design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/9.%20Download?preview=download-cloud-autopilot.png) and the [demo](http://www.ubuntu.com-1624-download-cloud-conjure-up.demo.haus/download/cloud/conjure-up)

## Issue / Card

Fixes #1624
